queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -48,7 +48,7 @@ def process(correction_filing: Filing, filing: Dict, filing_meta: FilingMeta, bu
             staff_id=correction_filing.submitter_id
         )
     )
-    if business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC']:
+    if business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC', 'CP']:
```

Comment:
So is this new filing a generic coop correction? Or only a correction to a SR?

Reply:
thanks for point it out, I updated to `or filing['correction']['correctedFilingType'] == 'specialResolution'`

Reply:
@argush3 any ideas?

Reply:
I think we should keep this generic as this is the general pattern up to this point.  i.e. add `CP` back to check on L51. 
 
This check should be done in the legal api validation code before it makes it to the filer.  @vysakh-menon-aot any opinions on this?

I think we should also verify with the business what COOP filings can be corrected.  I noticed that AR filings can be corrected right now.  If we add a check for allowed correction filing types in legal api validation for COOPs,  it will need to include at least special resolution and AR as a valid `correctedFilingType`.

Reply:
ok, interesting.  yes, look like AR corrections(and I imagine COA and COD corrections) reach the filer but just gets set with the status of "PENDING_CORRECTION.

I think you are right @severinbeauvais .  @Jxio ignore my previous comments and just leave your code(`or filing['correction']['correctedFilingType'] == 'specialResolution'`) as is.

Reply:
According to the business we need to be able to correct Special Resolutions and Corrections.